config/config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "development": {
+    "username": "postgres",
+    "password": null,
+    "database": "postgres",
+    "host": "localhost",
+    "dialect": "postgres",
+    "operatorsAliases": "false"
+  },
+  "test": {
```

Comment:
Comment (<--when I say that, I mean something non-blocking to the PR): We might want to consider using config overlays, either via secretly stashed files or ENV vars, in the future rather than calling out all the environments/DBs in there. i.e. one database is listed in the file, and in this repo it's setup as the localhost one.

Reply:
for sure, that was the ultimate plan, at the moment I just wanted it to work without too much extra hassle on the devs part. 

But ultimately it should read all this stuff from ENV vars on the local machine, this was just a simplification and a desire not to add *too* much stuff to this PR.